### DISCUSSION TITLE
3rd party OAuth access

### DISCUSSION THREAD

**Discussion by vteague** [DemocracyDevelopers/AuditVisualizer#28] [Announcements]
Hi everyone, especially @Linkgls , I got your request for 3rd party OAuth access for Notion. I need a bit more information though:
1. Since the repo is public, I'm not sure why we need to explicitly grant access to private data? Is that just github being confusing, or can we make a request for public data only?
2. Can you make a request specific to the visualiser repo, rather than all of the DD org? Thanks.

**Comment by Mirrorgo** [comment]
Got it, I'll remind my teammates to request only specific permissions. Thanks for the reminder!

**Reply by Linkgls** [reply]
Sry for the trouble that I've caused by my missteps on Notion. And we have already modified the URL display mode on Notion.
